docs/vm-ee-guide.md

```diff
@@ -0,0 +1,267 @@
+          f:nephe.io/cloud-vm-name:
+          f:nephe.io/cloud-vm-uid:
+          f:nephe.io/cloud-vpc-name:
+          f:nephe.io/cloud-vpc-uid:
+          f:nephe.io/kind:
+          f:nephe.io/namespace:
+          f:nephe.io/owner-vm:
+          f:nephe.io/owner-vm-vpc:
+      f:spec:
+        .:
+        f:endpoints:
+        f:externalNode:
+    Manager:         nephe-controller
+    Operation:       Update
+    Time:            2023-04-17T19:50:18Z
+  Resource Version:  556954
+  UID:               f0309a24-632b-4ebd-bbeb-9b205151a311
+Spec:
+  Endpoints:
+    Ip:           10.0.0.4
+    Ip:           20.245.116.185
+  External Node:  nephe-controller
+Events:           <none>
+```
+
+### Labels
+
+Labels with a `cloud-` prefix reflect the values fetched from cloud.
+
+| Label                   | Description                                              | Azure                                    | AWS                          |
+|-------------------------|----------------------------------------------------------|------------------------------------------|------------------------------|
+| nephe.io/cloud-vm-uid   | Unique identifier of VM                                  | `vmId` field on cloud                    | `Instance ID` of VM on cloud |
+| nephe.io/cloud-vpc-uid  | Unique identifier of VPC                                 | `resourceGuid` field on cloud            | `VPC ID` of VPC in cloud     |
+| nephe.io/cloud-vpc-name | User assigned VPC name on the cloud                      | `Name` field on cloud                    | `Name` tag                   |
+| nephe.io/cloud-vm-name  | User assigned VM name on the cloud                       | `Name` field on cloud                    | `Name` tag                   |
+| nephe.io/owner-vm       | VM object Name within Kubernetes cluster                 | vm `Name` and hash of vm resource `id`   | `Instance ID` of VM on cloud | 
+| nephe.io/owner-vm-vpc   | VPC object(belongs to VM) Name within Kubernetes cluster | vpc `Name` and hash of vpc resource `id` | `VPC ID` of VPC in cloud     | 
+
+
+nephe.io/cloud-region: Region configured in CPA and VMs are imported from this region.
+
+### Cloud Tags
+
+The tags on the `VirtualMachine` object are imported as labels in `ExternalEntity` CR. Label name starting with a prefix
+"tag-" are the `tags` from imported `VirtualMachine` objects. In the above example, `nephe.io/tag-Cloud` and
+`nephe.io/tag-Name` labels correspond to tags in `VirtualMachine` objects.
+
+#### Usage
+
+Labels in `ExternalEntity` can be directly used in To, From, AppliedTo section as externalEntitySelectors in Antrea
+`NetworkPolicys`. Following ANP demonstrates use of the label `nephe.io/tag-Name` in an `appliedTo` field.
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: crd.antrea.io/v1alpha1
+kind: NetworkPolicy
+metadata:
+name: vm-anp
+namespace: sample-ns
+spec:
+appliedTo:
+- externalEntitySelector:
+  matchLabels:
+    nephe.io/tag-Name: ubuntu-vm
```

Comment:
what does this translate too in the above sample output?

Reply:
I have a separate section for cloud tags right below the table, thats why didn't include in the table. Added now, let me know if this looks better or the earlier one.